frontend/src/data/shards/actions.js

```diff
@@ -0,0 +1,32 @@
+export const SHARD_ACTION_TYPES = {
+    LOAD_SHARD_STATUS: 'LOAD_SHARD_STATUS',
+    LOAD_SHARD_STATUS_SUCCESS: 'LOAD_SHARD_STATUS_SUCCESS',
+    LOAD_SHARD_STATUS_ERROR: 'LOAD_SHARD_STATUS_ERROR',
+};
+
+export function loadShardStatus() {
+    return async (dispatch) => {
+        dispatch({
+			type: SHARD_ACTION_TYPES.LOAD_SHARD_STATUS,
+		});
+
```

Comment:
Looks like there is some indentation issue here (1 too many + maybe indented with tabs?)

Reply:
Yikes for sure. Will have a quick swizz at what's happened there.